runtime/oti/j9nonbuilder.h

```diff
@@ -5540,8 +5547,7 @@ typedef struct J9VMThread {
 	UDATA contiguousIndexableHeaderSize;
 	UDATA discontiguousIndexableHeaderSize;
 #if defined(J9VM_ENV_DATA64)
-	UDATA isIndexableDataAddrPresent;
-	U_32 isVirtualLargeObjectHeapEnabled;
+	U_32 indexableObjectLayout; /* can be J9IndexableObjectLayout_NoDataAddr_NoArraylet,J9IndexableObjectLayout_DataAddr_NoArraylet, J9IndexableObjectLayout_NoDataAddr_Arraylet, J9IndexableObjectLayout_DataAddr_Arraylet */
```

Comment:
Either this comment should be shorter, some like "valid values are  #define J9IndexableObjectLayout_xyz"
or we don't  any comment here (as is the case with other fields) but we comment just above J9IndexableObjectLayout #defines that they are used in indexableObjectLayout field.


(same for J9JavaVM)